javascript/javascript2/week3/Game/script.js

```diff
@@ -0,0 +1,72 @@
+const btn = document.getElementById("startBtn");
+let inputTime = document.getElementById('inputTime');
+const timer = document.getElementById("timer");
+
+const scoreS = document.getElementById("scoreS");
+const scoreL = document.getElementById("scoreL");
+
+const divForS = document.getElementById("right");
+const divForL = document.getElementById("left");
+
+const wonSkey = document.getElementById("wonSkey");
+const wonLkey = document.getElementById("wonLkey");
+
+let startNUmS = 0;
+let startNUmL = 0;
+
+btn.addEventListener("click", () => {
+    let StartingSec = inputTime.value;
```

Comment:
Great job! I would suggest having a default value for starting sec, or that it would alert if the value is zero.